src/object/player.hpp

```diff
@@ -135,6 +135,18 @@ class Player final : public MovingObject
   bool is_invincible() const { return m_invincible_timer.started(); }
   bool is_dying() const { return m_dying; }
 
+  /**
+   * Returns true if the player is currently alive 
+   * (not dying or dead)
+   */
+  bool is_alive() const { return !is_dying() && !is_dead(); }
+  
+  /**
+   * Returns true if the player can be controlled
+   * by the player (alive and not currently in a win sequence)
+   */
+  bool is_active() const { return is_alive() && !is_winning(); }
```

Comment:
```suggestion
  /**
   * Returns true if the player can be controlled.
   * (alive and not currently in a win sequence)
   */
  bool is_active() const { return is_alive() && !is_winning(); }
```

I just think repeating the word "player" while meaning two different things doesn't sound well and could probably cause confusion.